core/server/api_container/server/startosis_engine/startosis_packages/git_package_content_provider/parsed_git_url.go

```diff
@@ -103,6 +103,11 @@ func parseGitURL(packageURL string) (*ParsedGitURL, *startosis_errors.Interpreta
 	return parsedGitURL, nil
 }
 
+func (parsedUrl *ParsedGitURL) getLocatorRelativeToThisURL(relativeUrl string) string {
```

Comment:
the method name says `relative` and the returned variable name says `absolute` is this ok?